example/centos/centos-9-aarch64-minimal-raw.yaml

```diff
@@ -0,0 +1,167 @@
+            - xfsprogs
+            - xz
+          exclude: []
+    os:
+      otk.external.osbuild-gen-depsolve-dnf4:
+        architecture: aarch64
+        module_platform_id: c9s
+        releasever: "9"
+        repositories:
+          otk.include: "common/repositories/aarch64.yaml"
+        packages:
+          include:
+            - "@core"
+            - NetworkManager-wifi
+            - dosfstools
+            - dracut-config-generic
+            - efibootmgr
+            - grub2-efi-aa64
+            - grub2-tools
+            - initial-setup
+            - iwl3160-firmware
+            - iwl7260-firmware
+            - kernel
+            - libxkbcommon
+            - selinux-policy-targeted
+            - shim-aa64
+            - xfsprogs
+          exclude: []
+  files:
+    otk.external.osbuild-gen-inline-files:
+      inline:
+        kickstart:
+          contents: |
+            # Run initial-setup on first boot
+            # Created by osbuild
+            firstboot --reconfig
+            lang en_US.UTF-8
+  kernel:
+    cmdline: ro
+    package:
+      otk.external.osbuild-get-dnf4-package-info:
+        packageset: ${packages.os}
+        packagename: "kernel"
+
+otk.include: "common/partition-table/aarch64/minimal-raw.yaml"
+
+otk.target.osbuild:
+  pipelines:
+    - otk.include: "pipeline/build/generic.yaml"
+    - name: os
+      build: name:build
+      stages:
+        - type: org.osbuild.kernel-cmdline
+          options:
+            root_fs_uuid: ${filesystem.const.partition_map.root.uuid}
+            kernel_opts: ${kernel.cmdline}
+        - otk.external.osbuild-make-depsolve-dnf4-rpm-stage:
+            packageset: ${packages.os}
+            gpgkeys:
+              otk.include: "common/gpgkeys.yaml"
+        - type: org.osbuild.fix-bls
+          options:
+            prefix: ''
+        - type: org.osbuild.locale
+          options:
+            language: C.UTF-8
+        - type: org.osbuild.timezone
+          options:
+            zone: America/New_York
+        - type: org.osbuild.sysconfig
+          options:
+            kernel:
+              update_default: true
+              default_kernel: kernel
+            network:
+              networking: true
+              no_zero_conf: true
+        - type: org.osbuild.systemd.unit
+          options:
+            unit: grub-boot-success.timer
+            dropin: 10-disable-if-greenboot.conf
+            config:
+              Unit:
+                ConditionPathExists: '!/usr/libexec/greenboot/greenboot'
+            unit-type: global
+        - otk.external.otk-make-fstab-stage:
+            ${filesystem}
+        - otk.include: fragment/grub2/aarch64.yaml
+        - type: org.osbuild.copy
+          inputs:
+            # XXX note we're only keeping the hash here since it corresponds to the reference
+            # XXX manifest and the same in the input
+            file-cbe2cb6e47e05af3b653e2567e9ee04ba7d9dcf8fdc2cfd15b232a0844c128f9:
```

Comment:
Silly question, could this be `file-${files.const.files.kickstart.id}` or something?